.github/workflows/clear-cache.yml

```
@@ -25,7 +25,7 @@ jobs:
     if: github.event_name == 'schedule' || github.event_name == 'pull_request'
     uses: Lightning-AI/utilities/.github/workflows/cleanup-caches.yml@v0.15.0
     with:
-      scripts-ref: v0.14.3
+      scripts-ref: v0.15.0
```

```suggestion
      scripts-ref: v0.14.3
```
reported in https://github.com/Lightning-AI/utilities/pull/371#issuecomment-3150211432